Makefile

```diff
@@ -82,7 +82,7 @@ Dockerfile.dev: Dockerfile Dockerfile.centos.patch
 	patch -o Dockerfile.dev Dockerfile Dockerfile.centos.patch
 
 image: Dockerfile.dev
-	echo podman build -f $^ -t $(IMAGE_TAG) .
```

Comment:
Shall we add back the `SKIP_BUILD` variable per the doc: https://github.com/openshift/elasticsearch-operator/blob/master/docs/HACKING.md#pushing-an-image-to-a-4x-cluster

Reply:
I've addressed this with the latest commit by using a real target.